src/Tizen.NUI/src/internal/Interop/Interop.BackgroundBlurEffect.cs

```diff
@@ -1,11 +1,49 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 namespace Tizen.NUI
 {
+    using global::System;
+    using global::System.Runtime.InteropServices;
+
     internal static partial class Interop
     {
-        internal static partial class BackgroundBlurEffect
+        internal static class BackgroundBlurEffect
         {
-            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_BackgroundBlurEffect_New__SWIG_1")]
-            public static extern global::System.IntPtr New(uint pixelRadius, bool blurOnce);
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_BackgroundBlurEffect_New__SWIG_1")]
+            public static extern IntPtr New(uint blurRadius);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_BackgroundBlurEffect_SetBlurOnce")]
+            public static extern void SetBlurOnce(HandleRef effect, bool blurRadius);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_BackgroundBlurEffect_GetBlurOnce")]
+            public static extern bool GetBlurOnce(HandleRef effect);
```

Comment:
```suggestion
            [return: MarshalAs(UnmanagedType.U1)]
            public static extern bool GetBlurOnce(HandleRef effect);
```

We should add `MarshalAs(UnmanagedType.U1)` keyword if binded function return boolean value.